lib/active_snapshot/config.rb

```diff
@@ -1,31 +1,50 @@
+# frozen_string_literal: true
+
 module ActiveSnapshot
   class Config
+    class InvalidStorageMethodError < StandardError
```

Comment:
This error occurs at boot time rather than runtime. So I do not think that we need a specific error class for this.